docs/seccomp.md

```diff
@@ -36,11 +36,16 @@ included in the respective release archive, viewable on the
 
 ## Custom filters (advanced users only)
 
-**Note**: This feature overrides the default filters and can be dangerous.
+**Note 1**: This feature overrides the default filters and can be dangerous.
 Filter misconfiguration can result in abruptly terminating the process or
 disabling the seccomp security boundary altogether.
 We recommend using the default filters instead.
 
+**Note 2**: The user is fully responsible for managing the filter files.
+We recommend using integrity checks whenever transferring/downloading files,
+for example checksums, just like with the Firecracker binary or other artifacts,
```

Comment:
Nit: For some reason, "just like with the Firecracker binary" sounds weird to me. Try maybe 
> for example checksums, *as well as for* the Firecracker binary or other artifacts,

or something similar.